packages/server/src/orchestrator/atom.ts

```diff
@@ -1,20 +1,18 @@
 import { Atom as _Atom, Slice as _Slice } from "@bigtest/atom";
 import { OrchestratorState } from "./state";
 
-export class Atom extends _Atom<OrchestratorState> {
-  constructor() {
-    super({
-      manifest: {
-        description: "None",
-        fileName: "<init>",
-        steps: [],
-        assertions: [],
-        children: [],
-      },
-      agents: {},
-      testRuns: {},
-    })
-  }
-}
+export const createOrchestratorAtom = () => new _Atom<OrchestratorState>({
+  manifest: {
+    description: "None",
+    fileName: "<init>",
+    steps: [],
+    assertions: [],
+    children: [],
+  },
+  agents: {},
+  testRuns: {},
+});
+
+export type Atom = _Atom<OrchestratorState>
```

Comment:
What are we gaining by aliasing `Atom` and `Slice` this way?

Reply:
There are a bunch of places where Atom is referenced. I would need to go through and change it all to `Atom<OrchestratorState>` this allows me to no do that. Do you want me to?

Reply:
I don't know what the best practice for TS, but it just seemed like something I've never seen before. I don't know if this is a code-smell or not?

Reply:
Not sure... I can apply `Atom<OrchestratorState>` to everything. It's 10 places. I think

Reply:
I have a slight preference for being explicit and not aliasing, but I don't have a strong opinion, so I'll leave  up to you.

Reply:
Removed the alias